docs/sdks/create-api-key.md

```diff
@@ -23,11 +23,14 @@ console.log('API Creation Failed: ', e);
 
 ```py
 from c8 import C8Client
+  
+API_ACTIVE = False
+CHOSEN_API_KEY_ID = "myNewKey"
+API_KEY = "my API key" # Change this to my key
+URL = "gdn.paas.macrometa.io"
+
+client = C8Client(protocol='https', host=URL, port=443, apikey = API_KEY)
 
```

Comment:
One doubt here since this is a create apikey example shouldn't we assume that the user doesn't have an apikey yet? Hence shouldn't the C8client login method be based on email and password rather than apikey for creating apikey example? Any thoughts?

Reply:
Yes. Assume that the user is creating an API key because they do not have one. Or we could just point them at the authentication topic and let them decide how to log in.

Reply:
ok, thanks guys.

Suggestions have been applied.